### PR TITLE
refactor broadcom-wl profile

### PR DESCRIPTION
**Pull request by vnepogodin**

### CODE REVIEW

File: profiles/pci/network_drivers/profiles.toml [hunk 1]
```diff
@@ -7,6 +7,19 @@ device_ids = "4311 4312 4315 4727 4328 4329 432A 432B 432C 432D 0576 4353 4357 4
...
+blacklist brcm80211
+blacklist brcmfmac
+blacklist brcmsmac
+blacklist bcma
+EOF
+    mkinitcpio -P
+
     modprobe -r b43
     modprobe -r b43legacy
     modprobe -r ssb
```
Comment (ventureoo) [inline profiles/pci/network_drivers/profiles.toml:25]: If modules are in use now, they will not be unloaded, so unloading them during `post_install` is useless

Comment (vnepogodin) [reply profiles/pci/network_drivers/profiles.toml:25]: So we just remove it, or move unload into pre_install?

Comment (ventureoo) [reply profiles/pci/network_drivers/profiles.toml:25]: > So we just remove it, or move unload into pre_install?

Remove it.

Comment (vnepogodin) [reply profiles/pci/network_drivers/profiles.toml:25]: Ok

**Pull request merged**